addons/website/static/src/js/utils.js

```diff
@@ -243,6 +243,20 @@ function websiteDomain(self) {
     return ['|', ['website_id', '=', false], ['website_id', '=', websiteID]];
 }
 
+/**
+ * @param  {Object} self
+ * @returns the website id.
+ */
+    function getWebsiteId(self) {
+    let websiteId;
+        self.trigger_up('context_get', {
```

Comment:
Don't create a method for this :) just simply add it in your code.
The trigger_up is already the util method, here you have an util method calling another util method

If we were to introduce this new helper, it should at bare minimum replace all the occurences doing that exact same code, including `websiteDomain()`

+ there is miss indentation